tick/stateful/eval_function_node.go

```diff
@@ -187,6 +216,12 @@ func eval(n NodeEvaluator, scope *Scope, executionState ExecutionState) (interfa
 		return n.EvalTime(scope, executionState)
 	case ast.TDuration:
 		return n.EvalDuration(scope, executionState)
+	case ast.TMissing:
```

Comment:
Do we need this? I can't think of a reason that we would want the result of an entire expression to evaluate to the type TMissing. Or am I misunderstanding the use of TMissing?